components/runtime/squeak5/patches/01-Makefile-amd64.patch

```diff
@@ -1,5 +1,5 @@
---- opensmalltalk-vm-sun-v5.0.7/platforms/unix/config/Makefile.in	Sat May  2 12:55:37 2020
-+++ p0/opensmalltalk-vm-sun-v5.0.7/platforms/unix/config/Makefile.in	Sat May  2 12:39:08 2020
+--- opensmalltalk-vm-sun-v5.0.8/platforms/unix/config/Makefile.in	Sat Jul  4 10:53:04 2020
++++ p0/opensmalltalk-vm-sun-v5.0.8/platforms/unix/config/Makefile.in	Sat Jul  4 10:17:13 2020
 @@ -77,10 +77,10 @@
```

Comment:
This change is not needed as only the path has changed (patch will do the right thing with the old path).